integration_test/third_party_apps_data/applications/sqlserver/metric_name.txt

```diff
@@ -0,0 +1 @@
+workload.googleapis.com/sqlserver.user.connection.count
```

Comment:
This file isn't used anymore and can be deleted. (The `representative: true` metric under `expected_metrics` replaces the old behaviour.)

Reply:
deleted